packages/vc-verification/src/issuer-verification.ts

```diff
@@ -188,7 +188,7 @@ export class IssuerVerification {
     );
     if (issuers && issuers.did && issuers.did.length > 0) {
       for (let i = 0; i < issuers.did.length; i++) {
-        if (issuers.did[i] == addressOf(issuerDID)) {
+        if (issuers.did[i] == issuerDID) {
```

Comment:
@JGiter The `issuers.did[I]` are addresses that comes from the `RoleResolvers`. So we have to compare the addresses in this scenario. Thus `addressOf` was being used to for IssuerDID.

Reply:
Or maybe we can generalise the role-resolvers to return DIDs instead of addresses?

Reply:
So you think that return type of `IssuerDefinitionResolver.getIssuerDefinition` should be not `IIssuerDefinition`?

Reply:
It should be there, but we should convert those `addresses` into DIDs before adding to the `IIssuerDefinition`. So, in general the issuers from any source ( Cache / ENS / something like VUI issuer interface ) would always return in the form of DIDs.

Reply:
Ok